ak_survey_results.py

```diff
@@ -23,6 +15,7 @@
     'DB_NAME': 'Database name',
     'DB_SCHEMA_AK': 'Database schema for ActionKit tables',
     'DB_SCHEMA_SURVEY': 'Database schema for survey results tables',
+    'DB_VARCHAR_COL': 'VARCHAR column type and length. VARCHAR on Postgres, VARCHAR(MAX) on Redshift',
```

Comment:
Hm, this seems...indirect.  We don't actually want to know the type to use as much as we want to know whether the DB is Redshift or Postgres.  Can we either detect the DB type automatically or pass that in as a parameter or environment variable instead?

Reply:
Actually, the DB type strikes me as the sort of thing that could be set in zappa_settings.py, since that's the file which is supposed to describe the production environment.  There'd have to be a fallback value for local testing though.

Reply:
This is an interesting idea. I did toy with the idea of detecting database type, but the database connection package is `psycopg2` which I think already assumes you are using Postgres. Anyway, I didn't finish that investigation because I moved on to the column type setting idea. I like your Zappa settings idea, though. I will look into that!

- [ ] Investigate using Zappa settings to indicate database type (and fall back to Postgres as local default)

Reply:
@sjwmoveon, I looked at the [Advanced Zappa Settings](https://github.com/zappa/Zappa#advanced-settings) and unfortunately didn't see anything that would communicate a database type. What do you think about changing the `settings` parameter to be `DB_TYPE` instead of `DB_VARCHAR_COL` and then set a variable for VARCHAR column type in the code based on the database type? I think that would be an improvement.

Reply:
That sounds good!  I was thinking of making DB_TYPE an environment variable but having it as a setting also works.

Reply:
- [ ] Replace DB_VARCHAR_COL setting parameter with DB_TYPE